frontend/src/components/events/organisms/event_section.css

```diff
@@ -8,3 +12,20 @@
   width: fill;
   margin: 0 14px;
 }
+
+.featured-event-top {
+  width: fill;
```

Comment:
fill is old -> please use `stretch` instead -> otherwise you will see a react es lint complaint about it